src/cf-pagination/usage.md

```diff
@@ -20,52 +22,55 @@ The cf-pagination component provides a responsive approach to multipage page nav
 
 ## Variables
 
-Theme variables for setting the color and sizes throughout the project. Overwrite them in your own project by duplicating the variable `@key: value`.
+Theme variables for setting the color and sizes throughout the project.
+Overwrite them in your own project by duplicating the variable `@key: value`.
 
 ### Color variables
 
-`$color-` variables are from 18F's [US Web Design Standards](https://github.com/18F/web-design-standards/blob/18f-pages/assets/_scss/core/_variables.scss)
+`$color-` variables are from 18F's
```

Comment:
You're coming up with even more reasons to get rid of these generic variables ;)